ledger-test-tool/infrastructure/src/main/scala/com/daml/ledger/api/testtool/infrastructure/LedgerTestSuite.scala

```diff
@@ -75,4 +88,71 @@ abstract class LedgerTestSuite {
   }
 
   def name: String = getClass.getSimpleName
+
+  def updateCommands(commands: JList[Command], f: CommandV1 => CommandV1): JList[Command] =
+    commands.asScala
+      .map(c => CommandV1.fromJavaProto(c.toProtoCommand))
+      .map(f)
+      .map(c => Command.fromProtoCommand(toJavaProto(c)))
+      .asJava
+
+  implicit class IdentifierConverter(id: JavaIdentifier) {
+    def toV1: Identifier = Identifier.fromJavaProto(id.toProto)
+  }
+
+  implicit def partyToString(party: Party): String = party.getValue
+
+  // TODO: when merged in canton (get it from ledger-common)
+  object TimestampConversion {
+    val MIN = Instant parse "0001-01-01T00:00:00Z"
+    val MAX = Instant parse "9999-12-31T23:59:59.999999Z"
+  }
+
+  object ClearIdsImplicits {
+
+    def clearIds(value: Value): Value = {
+      val sum = value.sum match {
+        case Sum.Record(record) =>
+          Sum.Record(
+            record.clearRecordId.copy(fields =
+              record.fields.map(f => f.copy(value = f.value.map(clearIds)))
+            )
+          )
+        case Sum.Variant(variant) =>
+          Sum.Variant(variant.clearVariantId.copy(value = variant.value.map(clearIds)))
+        case Sum.List(list) => Sum.List(ListV1(list.elements.map(clearIds)))
+        case Sum.Optional(optional) =>
+          Sum.Optional(Optional(optional.value.map(clearIds)))
+        case Sum.GenMap(genmap) =>
+          Sum.GenMap(GenMap(genmap.entries.map({ case GenMap.Entry(k, v) =>
+            GenMap.Entry(k.map(clearIds), v.map(clearIds))
+          })))
+        case Sum.Map(simplemap) =>
+          Sum.Map(MapV1(simplemap.entries.map({ case MapV1.Entry(k, v) =>
+            MapV1.Entry(k, v.map(clearIds))
+          })))
+        case _ => value.sum
+      }
+      Value(sum)
+    }
+
+    implicit class ClearValueIdsImplicits(record: com.daml.ledger.api.v1.value.Record) {
+      // TODO: remove when java bindings toValue produce an enriched Record w.r.t. type and field name
```

Comment:
Mind that you will need to create issues for these TODOs upon migrating to canton